corin_control/MotionPlan motionPlan
---
float64[] CoM
float64[] CRBI